py3status/parse_config.py

```diff
@@ -292,6 +311,42 @@ def make_value(self, value):
             return None
         return value
 
+    def make_value_from_env(self, value):
+        # Extract the environment variable and type
+        match = re.match('env\(\s*([0-9A-Z_]+)(\s*,\s*[a-zA-Z_]+)?\s*\)', value)
```

Comment:
Why not allow vars that are lower case?

'env\\(\s*([0-9**a-z**A-Z_]+)(\s*,\s*[a-zA-Z_]+)?\s*\\)'

Reply:
Sure! Fixed. :)